ci/docker/runtime_functions.sh

```diff
@@ -320,6 +320,8 @@ build_ubuntu_cpu_openblas() {
         -DUSE_BLAS=Open \
         -DUSE_ONEDNN=OFF \
         -DUSE_CUDA=OFF \
+        -DUSE_CPP_PACKAGE=ON \
+        -DMXNET_USE_CPU=ON \
```

Comment:
I'm not sure whether this build option makes much sense as mxnet always has cpu mode enabled. For the lenet example, it would be great to have the code detect the GPU count instead